python/tvm/relay/_parser.py

```diff
@@ -453,16 +466,23 @@ def visitIncompleteType(self, ctx):
         # type (RelayParser.IncompleteTypeContext) -> None:
         return None
 
-    def visitIdentType(self, ctx):
-        # type: (RelayParser.IdentTypeContext) -> Union[ty.TensorType, str]
-        ident_type = ctx.CNAME().getText()
+    def visitTypeIdent(self, ctx):
+        '''
+        Handle type identifier.
+        type: (RelayParser.TypeIdentContext) -> Union[ty.TensorType, str]
```

Comment:
this type annotation should be `# type: (RelayParser.IdentTypeContext) -> Union[ty.TensorType, str]` and go above and outside the doc string. cf: https://mypy.readthedocs.io/en/latest/cheat_sheet.html

Reply:
Pylint complained when I had it that way

Reply:
Ah, it complained because I had the annotation after the docstring, not before

Reply:
Done